### ISSUE TITLE
Add opentype profile VF check for nameID 1

### ISSUE THREAD

**Issue by davelab6** [googlefonts/fontbakery#2887]
@ebensorkin thought his VF was buggy, but it was correct, as discussed in https://github.com/googlefonts/fontmake/issues/666#issuecomment-630448415

Therefore we miight have a WARN check that, when the font is a VF and the nameID 1 is not the base family name (eg, it has style name particles, found elsewhere in the VF tables), we let users know this is expected, but they might want to pick a different master to use as the 'zero origin.' 

This also belongs as a small update to https://github.com/googlefonts/gf-docs/tree/master/Spec#font-zero-origin

**Comment by m4rc1e** [comment]
I don't think we need a check that produces a warning for this. Reading the ms spec [name table](https://docs.microsoft.com/en-us/typography/opentype/spec/name#name-ids) makes it quite clear what's expected for each nameID.

Improving all the name checks is a better idea since they are confusing.
